developer/scripts/win_build_system_teardown.cmd

```diff
@@ -7,4 +7,8 @@
 %USERPROFILE%\Miniconda3\Scripts\conda env remove -y --name py36_64
 %USERPROFILE%\Miniconda3\Scripts\conda env remove -y --name py37_32
 %USERPROFILE%\Miniconda3\Scripts\conda env remove -y --name py37_64
+%USERPROFILE%\Miniconda3\Scripts\conda env remove -y --name py38_32
+%USERPROFILE%\Miniconda3\Scripts\conda env remove -y --name py38_64
+%USERPROFILE%\Miniconda3\Scripts\conda env remove -y --name py39_32
+%USERPROFILE%\Miniconda3\Scripts\conda env remove -y --name py39_64

```

Comment:
Per https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#removing-an-environment I think you want the `--all` here too. Not likely critical but does a full cleanup.

Reply:
That link says either `remove --name NAME --all` or `env remove --name NAME`. This is using the `env remove` syntax. No indication in there of a `env remove --all` possibility.